lib/encryptor.rb

```diff
@@ -15,7 +15,10 @@ module Encryptor
   #
   # Run 'openssl list-cipher-commands' in your terminal to view a list all cipher algorithms that are supported on your platform
   def default_options
-    @default_options ||= { :algorithm => 'aes-256-cbc' }
+    @default_options ||= {
+        :algorithm => 'aes-256-cbc',
+        :hmac_algorithm => 'sha1',
```

Comment:
It's a minor point, ultimately, especially since it can be overridden manually.  Still, if it's supported widely enough, I'd prefer sha256.  It sounds like we're in agreement that strong defaults are the best bet.

Thanks for your work on this, BTW!